src/cpp/README.md

```diff
@@ -70,18 +70,26 @@ also sets up an `add_subdirectory()` rule for you. This causes gRPC to be
 built as part of your project.
 
 ```cmake
+cmake_minimum_required(VERSION 3.15)
+project(my_project)
+
 include(FetchContent)
 FetchContent_Declare(
   gRPC
   GIT_REPOSITORY https://github.com/grpc/grpc
   GIT_TAG        v1.25.0
 )
+set(FETCHCONTENT_QUIET OFF)
 FetchContent_MakeAvailable(gRPC)
 
-add_executable(my_exe my_exe.cc)
+add_executable(my_exe my_exe.cxx)
```

Comment:
why cxx and not .cc? (this repo uses .cc for C++ files)

Reply:
👍